frontend/src/components/settings/ManageUserDialog.vue

```diff
@@ -274,6 +288,82 @@ export default {
     getAddUserSuccessMsg() {
       return ADD_USER_SUCCESS_MSG
     },
+
+    /**
+     * Returns true if users facility access has changed, false otherwise.
+     */
+    hasUserFacilityAccessChanged(selectedFacilityIds, userFacilities) {
+      if (selectedFacilityIds?.length !== userFacilities?.length) {
+        return true;
+      } else {
```

Comment:
There's no need for an else block since the if block returns.

Reply:
Fixed.